myfcn.C

```diff
@@ -0,0 +1,122 @@
+/*
```

Comment:
This is the way I define the chi2 functions, i.e. the combine function and its Eval() method, and compile them in the respective Sim*.py macro. 
(I have to change it to a more appropriate name, plus to edit its content, because currently its confusing indeed)